cli/src/main/scala/filodb.cli/CliMain.scala

```diff
@@ -266,7 +266,7 @@ object CliMain extends ArgMain[Arguments] with FilodbClusterNode {
   def parseLabelValuesQuery(client: LocalClient, labelNames: Seq[String], constraints: Map[String, String], dataset: String,
                             timeParams: TimeRangeParams,
                             options: QOptions): Unit = {
-    val logicalPlan = LabelValues(labelNames, constraints, 3.days.toMillis)
+    val logicalPlan = LabelValues(labelNames, constraints, timeParams.start * 1000, timeParams.end * 1000)
```

Comment:
Can you please don't break the existing behavior. If `timeParams.start` is not set by user, fallback to default `3.days.tomillis` window.